src/shippingservice/main.go

```diff
@@ -58,8 +59,36 @@ func init() {
 }
 
 func main() {
-	go initTracing()
-	go initProfiling("shippingservice", "1.0.0")
+	stats, err := getenvBool("STATS")
+	if err != nil {
+		log.Errorf("Could not get STATS var: %v, defaulting to True", err)
+		stats = true
+	}
+
+	trace, err := getenvBool("TRACE")
+	if err != nil {
+		log.Errorf("Could not get TRACE var: %v, defaulting to True", err)
+		trace = true
+	}
+
+	profiler, err := getenvBool("PROFILER")
+	if err != nil {
+		log.Errorf("Could not get PROFILER var: %v, defaulting to True", err)
+		profiler = true
+	}
```

Comment:
imagine just doing

    profilerEnabled := os.Getenv("DISABLE_PROFILER") == ""

no parsing methods, no errors, no error handling, no boolstrings in yaml

Reply:
thanks for this! you're totally right, for some reason I thought it was important to check true/false, but simply checking if the env is set is a much better option. updated all the services to remove bool parsing.